### PR TITLE
Implement app banner new dismiss button UX

### PR DESCRIPTION
**Pull request by mkhatib**
cc/ @abeck000

screen shot 2016-10-26 at 3 53 06 pm

localhost-8000-examples-viewer html chrome ios 1

### CODE REVIEW

**Comment by ericlindley-g** [comment]
LGTM — deferring to @jasti and @zhouyx for their approval as well

File: extensions/amp-app-banner/0.1/amp-app-banner.css [hunk 1]
```diff
@@ -19,44 +19,61 @@ amp-app-banner {
...
   left: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 12px;
+  background-position: 6px center;
 }
 
 [dir=rtl] .amp-app-banner-dismiss-button:before {
-  top: -18px;
-  right: -18px;
+  top: -20px;
```
Comment (zhouyx) [inline extensions/amp-app-banner/0.1/amp-app-banner.css]: same top value,  not needed.

File: extensions/amp-app-banner/0.1/amp-app-banner.css [hunk 2]
```diff
@@ -19,44 +19,61 @@ amp-app-banner {
...
-  background-size: 26px 26px;
-  background-position: center;
+  background-image: url('data:image/svg+xml;utf8,%3Csvg%20width%3D%2213px%22%20height%3D%2213px%22%20viewBox%3D%22341%208%2013%2013%22%20version%3D%221.1%22%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20xmlns%3Axlink%3D%22http%3A//www.w3.org/1999/xlink%22%3E%3Cg%20id%3D%22ic_close_black_24dp%22%20stroke%3D%22none%22%20stroke-width%3D%221%22%20fill%3D%22none%22%20fill-rule%3D%22evenodd%22%20transform%3D%22translate%28341.000000%2C%208.000000%29%22%3E%3Cpolygon%20id%3D%22Shape%22%20fill%3D%22%234F4F4F%22%20points%3D%2213%201.30928571%2011.6907143%200%206.5%205.19071429%201.30928571%200%200%201.30928571%205.19071429%206.5%200%2011.6907143%201.30928571%2013%206.5%207.80928571%2011.6907143%2013%2013%2011.6907143%207.80928571%206.5%22%3E%3C/polygon%3E%3C/g%3E%3C/svg%3E');
+  background-size: 13px 13px;
+  background-position: 9px center;
   background-color: #fff;
-  border: 1px solid #c8c8c8;
+  background-repeat: no-repeat;
   z-index: 13;
```
Comment (zhouyx) [inline extensions/amp-app-banner/0.1/amp-app-banner.css]: think about it, should we add `!important` to z-index?

**Comment by mkhatib** [comment]
@zhouyx yes, we decided that it's too hard to control this. Although my change make use of a private element to create the `white` 4px height bar on the top of the banner - which basically makes it unstylable. However the dismiss button itself is still stylable.

**Comment by jasti** [comment]
Thanks! LGTM. Agreed that it's too hard restrict someone from changing it. We just want to make sure that the default implementation looks good.

**Comment by jridgewell** [comment]
Is the app-banner fixed position at the top? If so, adding the button to the top looks really ugly.

**Comment by mkhatib** [comment]
@jridgewell the banner is fixed to the bottom of the page with max-height of 100px.

File: extensions/amp-app-banner/0.1/amp-app-banner.css (hunk 1 above)
Comment (mkhatib) [inline extensions/amp-app-banner/0.1/amp-app-banner.css]: Done

**Comment by jridgewell** [comment]
Oh, they're two separate images. :doh:

**Pull request merged**